test/new-e2e/pkg/utils/clients/ssh.go

```diff
@@ -1,399 +0,0 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-present Datadog, Inc.
-
-package clients
-
-import (
-	"bytes"
-	"errors"
-	"fmt"
-	"io"
-	"io/fs"
-	"net"
-	"os"
-	"path"
-	"time"
-
-	"github.com/cenkalti/backoff"
-	"github.com/pkg/sftp"
-	"golang.org/x/crypto/ssh"
-	"golang.org/x/crypto/ssh/agent"
-)
-
-// GetSSHClient returns an ssh Client for the specified host
-func GetSSHClient(user, host string, privateKey, privateKeyPassphrase []byte, retryInterval time.Duration, maxRetries uint64) (client *ssh.Client, err error) {
```

Comment:
Note for reviewers: this moved to https://github.com/DataDog/datadog-agent/pull/25346/files#diff-31b475ed96a35c5360a579e694f9d6fd55b88dc75420a2ae2dad6a4ade071544R28